app/generators/index.html

```diff
@@ -5,7 +5,27 @@
 class: discovery-page
 ---
   <section class="main">
-    <small>Your generator must have a GitHub repository description, the `yeoman-generator` keyword and a description in `package.json` to be listed. Official generators are marked with <img src="/assets/img/mustache.svg" width="20px"></small>
+    <a id="instructions-toggle">How to add your generator &raquo;</a>
+    <div id="instructions" style="display:none">
+      <p>
+        Your generator will automatically show up in the list if it meets the following criteria. The list is populated every hour.
+      </p>
+      <ul>
+        <li>Be published on the npmjs.org repository</li>
+        <li>Have the <code>yeoman-generator</code> keyword</li>
+        <li>Have a description in the <code>package.json</code> file or in its associated Github repository</li>
```

Comment:
Drop the mention of GitHub description. We support package.json description now, so we should just prefer that.